src/common/__tests__/__snapshots__/dateHelpers.test.js.snap

```diff
@@ -46,6 +51,333 @@ Object {
+      "value": Object {
+        "endDate": 2019-07-31T23:59:59.999Z,
+        "startDate": 2019-07-01T00:00:00.000Z,
+      },
+    },
+    "december": Object {
+      "_title": "december",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"December 2018\\"})",
+      "value": Object {
+        "endDate": 2018-12-31T23:59:59.999Z,
+        "startDate": 2018-12-01T00:00:00.000Z,
+      },
+    },
+    "february": Object {
+      "_title": "february",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"February\\"})",
+      "value": Object {
+        "endDate": 2019-02-28T23:59:59.999Z,
+        "startDate": 2019-02-01T00:00:00.000Z,
+      },
+    },
+    "january": Object {
+      "_title": "january",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"January\\"})",
+      "value": Object {
+        "endDate": 2019-01-31T23:59:59.999Z,
+        "startDate": 2019-01-01T00:00:00.000Z,
+      },
+    },
+    "july": Object {
+      "_title": "july",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"July\\"})",
+      "value": Object {
+        "endDate": 2019-07-31T23:59:59.999Z,
+        "startDate": 2019-07-01T00:00:00.000Z,
+      },
+    },
+    "june": Object {
+      "_title": "june",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"June\\"})",
+      "value": Object {
+        "endDate": 2019-06-30T23:59:59.999Z,
+        "startDate": 2019-06-01T00:00:00.000Z,
+      },
+    },
+    "march": Object {
+      "_title": "march",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"March\\"})",
+      "value": Object {
+        "endDate": 2019-03-31T23:59:59.999Z,
+        "startDate": 2019-03-01T00:00:00.000Z,
+      },
+    },
+    "may": Object {
+      "_title": "may",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"May\\"})",
+      "value": Object {
+        "endDate": 2019-05-31T23:59:59.999Z,
+        "startDate": 2019-05-01T00:00:00.000Z,
+      },
+    },
+    "november": Object {
+      "_title": "november",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"November 2018\\"})",
+      "value": Object {
+        "endDate": 2018-11-30T23:59:59.999Z,
+        "startDate": 2018-11-01T00:00:00.000Z,
+      },
+    },
+    "october": Object {
+      "_title": "october",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"October 2018\\"})",
+      "value": Object {
+        "endDate": 2018-10-31T23:59:59.999Z,
+        "startDate": 2018-10-01T00:00:00.000Z,
+      },
+    },
+    "september": Object {
+      "_title": "september",
+      "title": "t(curiosity-toolbar.granularityRange, {\\"context\\":\\"September 2018\\"})",
+      "value": Object {
+        "endDate": 2018-09-30T23:59:59.999Z,
+        "startDate": 2018-09-01T00:00:00.000Z,
+      },
+    },
+  },
+  "listDateTimeRanges": Array [
+    Object {
+      "_title": "july",
```

Comment:
optional:
may want to change this `_title` to `current`

Reply:
can go either way on this, using `current` does feel more useful, means if we had to we won't have to know the "current" month, we'll update it, but again, we'll have to start using the field to fully realize limitations